app_levels/fr.json

```diff
@@ -1,13 +1,13 @@
 {
-    "applevel_0":  "Cassé",
-    "applevel_1":  "Installable",
-    "applevel_2":  "Installable dans toutes les situations",
-    "applevel_3":  "Peut être mis à jour",
-    "applevel_4":  "Supporte l'authentification unique",
-    "applevel_5":  "Propre",
-    "applevel_6":  "Supporte la sauvegarde et la restauration",
-    "applevel_7":  "Réussit tous les tests fonctionnels",
-    "applevel_8":  "Respecte les principales recommandations",
-    "applevel_9":  "Respecte toutes les recommandations",
-    "applevel_10": "Package jugé parfait"
+    "applevel_0": "Cassée",
+    "applevel_1": "Installable",
+    "applevel_2": "Installable dans toutes les situations",
+    "applevel_3": "Peut être mise à jour",
+    "applevel_4": "Supporte l’authentification unique",
+    "applevel_5": "Propre",
+    "applevel_6": "Supporte la sauvegarde et la restauration",
+    "applevel_7": "Réussit tous les tests fonctionnels",
+    "applevel_8": "Respecte les principales recommandations",
+    "applevel_9": "Respecte toutes les recommandations",
+    "applevel_10": "Paquet jugé parfait"
```

Comment:
Package...

Reply:
Can we please have feedback from @frju365 @Psycojoker @alexAubin @M5oul etc?

Reply:
Paquet ne me semble pas gênant ici, puisqu'on dit paquet debian*, or ces paquets ressemblent bien aux paquets debian (du moins dans un certain sens/objectif).
Par contre, le nom générique packaging, le nom du professionnel/métier appliqué au nom précédent packager/packageurs et le verbe packager resteront car il sont plus cours, et concernent l'informatique (empaqueter me semble très lourd dans ce cas) 

*On ne dit pas : "package debian" mais bien "paquet debian". À l'inverse paquet sera anglicisé dans un langage plus "franglish" mais l'ordre adj/nom sera inversé : "debian package"

Reply:
On parle toujours de package et de packagers, ainsi que de packaging.
Soyons cohérent...

Reply:
Euh, quand je parle à qqn sur le forum ou sur le chat, je parle de paquet. Et non, ce n'est pas cohérent. Le débutant n'utilisera jamais packaging et packager, par contre il utilisera beaucoup le terme de paquet. On dit un paquet debian, pas un package debian. Je suis vraiment désolé, mais je parle par expérience d'utilisation. Toujours est-il que "cassée" est incohérent ici.

Reply:
Je reste sur ma position. La cohérence a ici à voir avec le public auquel on s'adresse. Les dev utilisent bcp plus packaging et packager. Rares sont les débutants qui disent packaging et packager (même l'utilisateur expérimenté...).

Reply:
Le débutant il utilisera le terme qu'on utilise.
Les packages YunoHost ça existait pas avant qu'on en fasse, donc ça prend le nom qu'on lui donne.
Qui a toujours été "package", ce n'est pas moi qui l'ai inventé, et quand j'ai débuté ici, je parlais de package, car c'est ainsi que les autres l'appelait.
Donc là, on rentre dans le délire Français habituel de vouloir toujours tout franciser.
Ça me semble pas franchement nécessaire de maintenir cette habitude si française...

Reply:
Par contre, les débutants vont peut-être vouloir vérifier qu'est-ce que les niveaux des applications signifient.

Reply:
C'est pas une question d'intuitif ou pas, si j'invente quelque chose et que je te dis que c'est un "Blob", c'est un Blob, intuitif ou pas. Le nom initialement donné défini l'objet, c'est ainsi pour toute chose.
Ce qui est contre-intuitif, c'est de changer le nom d'un objet par une traduction de ce même nom.
Le Français est ma langue, malheureusement ces derniers temps..., je laisse le soin aux autres de se préoccuper de leur langues.
Mais il me semblerais bien plus simple pour tout le monde de ne pas traduire les noms, quelque soit la langue...

Reply:
Ouais... je ne comprends pas du tout ton raisonnement, et en quoi Yunohost a inventé le mot "package" ?! On parle de paquet sur le forum, on ne parle pas de "package" (à part toi)... donc, en tant que bon modérateur et en suivant toujours ton idée, il faudrait les reprendre (tout le monde), puisque apparemment ce "paquet" n'existe pas, ou serait incorrect, puisque un "package", c'est un "package". Sérieusement, ce n'est pas du tout la manière dont je vois l'internationalisation du projet. Package est un nom commun que tu le veuilles ou non... selon moi, il peut donc être traduit (si bien-sûr on ne perd pas son sens originel). Pour packageur et packaging, en effet, le traduire serait incorrect, puisque les termes français de ces nom, ne sont pas pas utilisé dans le domaine informatique.

J'aimerais bien avoir les points de vue de tout le monde. 
ping @YunoHost/apps @YunoHost/core-dev @YunoHost/doc

Reply:
Je ne dis pas que YunoHost a inventé le mot, mais le concept de package avec des scripts tels qu'on le fait est une construction propre à YunoHost. Même si le concept global ne l'est pas.
Cet ensemble de scripts pour installer une app a toujours été appelé un package, et lorsque je suis arrivé c'est ainsi qu'on me l'a défini. Et c'est aussi ainsi dont tout le monde en parlait, sur le forum ou le chat, débutant ou pas.
Et cela jusqu'à ce qu'un chantre de la langue française vienne hurler au scandale pour notre usage d'angliscismes. Et dés lors tente d'imposer sa vision d'une langue française pure et non souillée de mots étranger... Et malheureusement oui, à force d'insister, certains usages sur le forum ont changés.
Et comme je ne supporte pas cette attitude franco-française consistant à tout traduire, je résiste pour que nous gardions le nom original des packages, tel qu'il était avant qu'on tente de le changer.

On me diras que c'est une vendetta personnelle, ce qui me passe bien au-dessus..., mais sans l'intervention du dit chantre, toute cette polémique ne serait pas et personne ne se poserait la question de traduire le nom donné aux packages YunoHost...

Reply:
Ton raisonnement ne me convainc pas. Même si l'approche a été celle-ci, il n'y en a aucune trace sur le forum, or la langue est faite pour se faire comprendre des autres aujourd'hui. Cependant, package me semble plus apporté la confusion que la compréhension. Package a été à la base trouvé pour se rapprocher des paquets/packages debian. Or à ce jour, on dit paquet Debian et pas package. Et non, package est un mot anglais qui n'a aucune raison de rester anglais dans une phrase française. Je ne suis pas du tout comme tu sembles le dire, en train de faire un croisade pour le français. Ici, on est dans un fichier de traduction, qui a la nécessité d'être traduit si le mot est plus facile à comprendre.

Comme je l'ai dit, avoir les avis de tout le monde serait plus intéressant.

Reply:
Je ne parlais pas de toi, mais de Jean-Baptiste, qui dés son arrivé à lancé une croisade contre de très nombreux mots anglais utilisé au sein de YunoHost.
Me concernant, je resterais ferme sur ce point, et n'utiliserais jamais une traduction française pour nommer les packages, qui ont toujours porté ce nom ici.
Utiliser des noms différents selon la langue, n'est certainement pour aider à la conpréhension collective...

Reply:
Si c'était une marque déposée ou un nom propre, je serais d'accord avec toi, mais là, dire paquet est quand même monnaie courante et c'est un nom propre. Le traduire me semble logique.

J'avais bien compris que la personne dont tu parlais était Jean-Baptiste.

Reply:
C'est la même hérésie que "courriel", "pourriel" ou autre stupidités de la langue française.
On a TOUJOURS parlé de package pour les applications YunoHost, pourquoi devrions nous maintenant changer pour une putain de traduction française !!!?

Reply:
parce que dans les autres langues, le mot est traduit et que le mouvement de la communauté va dans le sens de l'utilisation de "paquet". Je suis même prêt à ouvrir un sondage sur le forum.

Pour les stupidité, non, ce n'est pas stupide, c'est la même chose pour toutes les langues. Il y a des traductions, c'est d'ailleurs pour cela qu'il y a plusieurs langues et pas un seule qui serait l'anglais. C'est vraiment stupide par contre de ne pas traduire un nom comme ça.Je ne vois pas pourquoi la langue d'autorité serait l'anglais pour des traduction française. C'est même aberrant.

Reply:
C'est une stupidité parce qu'on a pas besoin de TOUT traduire en permanence, si un mot défini quelque chose, on en a rien à foutre de savoir si le mot est français, anglais ou quoi que ce soit d'autre !
Mais on a ce problème uniquement en français, car on veut pas voir notre langue envahi par des mots étranger !
Et c'est pour cette raison que je quitte de plus en plus cette langue qui s'enferme de plus en plus !

So, let's do that a simple way. FUCK FRENCH LANGUAGE !!!
I'm not using it anymore on my own YunoHost anyway since a few month, because of those stupid french considerations.
So you want to keep french for french people with only french word and no bad foreign words, which are probably going to kill french if nobody stop them. Do that... I'm not fighting anymore against that stupid language which can't evolve.
Fuck it.

Reply:
Maniack: C'est pareil dans les autres langues. La preuve, prenons email :
anglais / français (car ça existe en français) : email
Français : courriel
Espagnol : correo electrónico
Polonais : poczta elektroniczna
Allemand : elektronische Post (E-Post)

Dans la plupart de ces langues email peut être utilisé bien-sûr.

Si tu vas par là, on peut utiliser les mots de n'importe quelle langue puisque chacun a un sens. CE N'EST PAS TOUT TRADUIRE MAIS TRADUIRE À BON ESCIENT, ET LÀ C'EST À BON ESCIENT PUISQUE TU ES LE SEUL À UTILISER LE MOT PACKAGE EN FRANÇAIS.